<launch>
  <arg name="bag_name" default="data" />
  <include file="$(find gait_training_robot)/launch/test5_comkf.launch" >
    <arg name="waypoints_file_suffix" value="_cw"/>
    <arg name="bag_name" value="$(arg bag_name)" />
  </include>
  <node name="foot_pose_estimator" pkg="gait_training_robot" type="foot_pose_estimator"/>
	<node name="record_monitor" pkg="gait_training_robot" type="record_monitor.py" output="screen"/>
</launch>